application/web/index.html

```diff
@@ -99,12 +100,27 @@ <h2 id="journal-title"></h2>
                 </div>
 
                 <!-- Temporarily put textarea for the CodeMirror editor -->
-                <textarea class="entry-textarea" placeholder="Enter your thoughts here"></textarea>
+                <!-- <textarea id="entry-textarea" class="entry-textarea" placeholder="PEE your thoughts here"></textarea> -->
+                <div id="entry-textarea" class="entry-textarea"></div>
                 <div class="past-entries"></div>
+                
+                <!-- TODO -->
+                <!-- Make this not be here, this should be a single include, that is included with the project -->
+                
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/6.65.7/codemirror.min.js"></script>
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/6.65.7/mode/markdown/markdown.min.js"></script>
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/6.65.7/mode/gfm/gfm.min.js"></script>
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.7/addon/mode/overlay.min.js"></script>
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.7/mode/meta.min.js"></script>
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.7/mode/xml/xml.min.js"></script>
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.7/mode/java/java.min.js"></script>
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.7/mode/python/python.min.js"></script>
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.7/mode/javascript/javascript.min.js"></script>
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.7/mode/brainfuck/brainfuck.min.js"></script>
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.7/mode/go/go.min.js"></script>
+                <script src="scripts/code-mirror-script.js"></script>
```

Comment:
can you put this first instead of last and see if it fixes it?

Reply:
like 121 to the top of the scripts?

Reply:
yeahh

Reply:
nah didn't work